test/surface/formatter_test.exs

```diff
@@ -1000,6 +1000,12 @@ defmodule Surface.FormatterTest do
       )
     end
 
+    test ":hook directive without any attribute" do
+      assert_formatter_doesnt_change("""
+      <div :hook />
+      """)
+    end
```

Comment:
This would have been great to add to the formatter in [surface-ui/surface](https://github.com/surface-ui/surface) itself :)

Care to put in another PR?

Reply:
sure, I am glad to add this to another PR